shared/opts-parser/opts-parser.c

```diff
@@ -34,6 +34,10 @@ struct obs_options obs_parse_options(const char *options_string)
 	size_t input_option_count = 0;
 	for (char **input_word = input_words; *input_word; ++input_word)
 		input_option_count += 1;
+	if (!input_option_count) {
```

Comment:
Nit: add a blank line before this to separate it from the un-enclosed `if` above.

Reply:
Should I add the blank line *before* the if statement? The if statement I've added is checking the result `input_option_count` so it might be better to add a blank line after the if closure.

Reply:
Sorry, I meant "unenclosed `for` above".

Reply:
So yes, add the blank line before the `if` statement.

Reply:
Added brank lines both before and after the if closure. I hope this should befine.